app/(tabs)/index.tsx

```diff
@@ -1,11 +1,74 @@
-import { View, Text } from "react-native";
+import { View, Text, StyleSheet, ScrollView } from "react-native";
+import ConfirmationCard from "../components/confirmationCard";
+import AppointmentCard from "../components/appointmentCard";
 
 const StartPage = () => {
     return (
-        <View>
-            <Text>Home Page</Text>
-        </View>
+        <ScrollView>
+            <View style={styles.greeting}>
+                <Text style={styles.greetingText}>Hallo Stephan!</Text>
+            </View>
+            <View style={{height: 40}}/>
+            <View style={styles.nextSteps}>
+                <Text style={styles.nextStepsHeadline}>Deine nächsten Schritte</Text>
+                <Text style={styles.nextStepsSub}>Alle Fortschritte anzeigen</Text>
+            </View>
+            <ScrollView style={styles.confirmationCardScrollView} horizontal showsHorizontalScrollIndicator={false}>
```

Comment:
When I scroll to the appointment cards there is no space between the last card and the border of the screen, which looks quite weird. I even think the card might be overflowing the viewport.